### DISCUSSION TITLE
Webpack issues starting up a project with nextjs, chakra UI. I don't know exactly what the issue is

### DISCUSSION THREAD

**Discussion by IbrahimIjai** [rainbow-me/rainbowkit#1406] [Feedback]
https://github.com/IbrahimIjai/ethereum-starter-project

Please help me check🙏🥲

**Comment by 0xmkh** [comment]
@IbrahimIjai Can you look at our example app here ? https://github.com/rainbow-me/rainbowkit/tree/main/examples/with-next-app

**Reply by 0xmkh** [reply]
Let me know if this doesn't work
